docs/plugins/repository-gcs.asciidoc

```diff
@@ -11,19 +11,19 @@ include::install_remove.asciidoc[]
 ==== Getting started
 
 The plugin uses the https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-clients/google-cloud-storage[Google Cloud Java Client for Storage]
-to connect to the Storage service. If you are using  
-https://cloud.google.com/storage/[Google Cloud Storage] for the first time, you 
+to connect to the Storage service. If you are using
```

Comment:
Your Atom setup is likely removing trailing whitespace when you save a file.

Not a huge issue, but you may want to consider turning this off. It can create noise in diffs and PRs.

Reply:
Cool, figuring out how to do this in Sublime.